packages/blob/src/index.ts

```diff
@@ -257,12 +257,13 @@ export async function list(
 }
 
 function getApiUrl(pathname = ''): string {
-  const baseUrl =
-    process.env.VERCEL_BLOB_API_URL ||
-    process.env.NEXT_PUBLIC_VERCEL_BLOB_API_URL ||
-    'https://blob.vercel-storage.com';
-
-  return `${baseUrl}${pathname}`;
+  let baseUrl = null;
```

Comment:
Could you add a comment here as the reasons behind this will be quite hard to understand based on the code alone

Reply:
@tcc-sejohnson could you double check this since you also prepared vite for edge config

Reply:
as @vvo suggested, we could also suggest to add `define : { process.env : {} } ` to the vite config, but I think we can avoid it as the variables are used only for the local testing

Reply:
I wonder why we need the try/catch in the first place. Could you instead check for `process?.env?.XXX`?

In Edge Config we have a note in the README but I see that it might not make sense for you https://github.com/vercel/storage/blob/main/packages/edge-config/README.md#a-note-for-vite-users. I'm not that familiar with Vite but @tcc-sejohnson fixed this in Edge Config and might have some pointers for you too

Reply:
the note is valid for a server access, in the client Vite injects the `import.meta.env.VITE_ABC` object.
I'm not sure how the bundlers are handling the process variables (i.e. the full `process.env.NEXT_PUBLIC_VERCEL_BLOB_API_VERSION_OVERRIDE` variable is defined, but if you try to access it dynamically `process.env[key]` it is not, also the process.env object is `{}`). So asserting on the process seems dangerous